content/recipes/interoperability/c4c-clinical-trials.md

```diff
@@ -0,0 +1,302 @@
+
+
+## Reviewing existing clinical trials registries
+
+
+The first step in the process defined and then refined a list of variables to be collected. The (advanced) search features of the following repositories and registries were recorded and then mapped to create a list of common metadata items across all of the resources:
+
+- [ClinicalTrials.gov](https://clinicaltrials.gov/)
+- [EU Clinical Trials Register](https://www.clinicaltrialsregister.eu/)
+- [Clinical Research Metadata Repository](https://ecrin.org/clinical-research-metadata-repository)
+- [Vivli](https://vivli.org/)
+- [The YODA Project](https://yoda.yale.edu/)
+- [Pediatric Trials Network (PTN)](https://pediatrictrials.org/)
+- [BioLINCC](https://biolincc.nhlbi.nih.gov/studies/)
+- [ClinicalStudyDataRequest (CSDR)](https://www.clinicalstudydatarequest.com/)
+- [Health Data Research Innovation Gateway](https://www.healthdatagateway.org/)
+- [NIDDK Central Repository](https://repository.niddk.nih.gov/search/study/)
+- [Project Data Sphere](https://data.projectdatasphere.org/projectdatasphere/html/access)
+- [Immunology Database and Analysis Portal (ImmPort)]()
+- [ITN TrialShare](https://www.immport.org/shared/)
+- [National Sleep Research Resource (NSRR)](https://sleepdata.org/)
+- [National Institute on Drug Abuse NIDA](https://datashare.nida.nih.gov/data)
+- [RDCA-DAP](https://portal.rdca.c-path.org/)
+- [European Genome-phenome Archive (EGA)](https://ega-archive.org/)
+
+The first step created a list of metadata items from the Advanced Search screen on ClinicalTrials.gov. Metadata items from each subsequent repository were compared against this list and mapped across if there was a match. For example, ‘Age Group’ and ‘Age Range’ or ‘Trial Phase’ and ‘Study Phase’. New metadata items that couldn’t be mapped against existing entries were added to the bottom of the list. This gave a good visual of which metadata items occurred most frequently across all of the repositories. The results of the mapping exercise were captured in a Google Sheet. We started with ClinicalTrials.gov because it is the most comprehensive and used repository.
+
+
+
+## Refining the initial metadata list
+The list of metadata items was reviewed by c4c partners, and those not considered cross cutting or common enough for paediatric clinical trials were removed from the list. The original list consisted of 36 items and this was reduced to 27. The following were identified for inclusion:
+
+| Considered cross cutting or common enough to be included in the metadata schema | Not considered cross cutting or common enough to be included in the Metadata Schema|
+| :------------- | :------------- |
+| Study IDs | Rare Disease (tick box yes/no) |
+| Title | Intervention Model |
+| Acronym | NCT Number |
+| Condition or Disease | Consent |

```

Comment:
strange that "consent" information is not included. any thing about "allowed use" of study data?

Reply:
I'm waiting to hear back from the c4c team but if memory serves, consent was excluded because it only appeared as a variable in one out of all the registries (BioLINCC). Since the primary goal of the profile is to be cross-compatible with as many registries as possible in order to represent metadata from historic clinical trials, singular variables were generally excluded. Consent is also quite difficult to model as it varies by legal framework.

Reply:
I agree consent is a relevant topic, but on the other hand it can be argued that consent is relevant for any dataset that contains patient data. I know ontologies like [Data Use Ontology](https://obofoundry.org/ontology/duo.html) and [Open Digital Rights Language](https://www.w3.org/TR/odrl-model/) can play a role here, but this would need more thought, and I would (for now) consider this out of scope for trial-specific metadata.

Reply:
Echoing what the others said, the legal framework around concept means there may be multiple definitions (e.g., for paediatrics is consent required from one or both parents). Mapping items that do not mean the same thing can be problematic.